packages/driver/cypress/e2e/commands/xhr.cy.js

```diff
@@ -1,2892 +1,38 @@
-        const url = 'http://localhost:3500/fixtures/generic.html'
-
-        const $a = win.$(`<a href='${url}'>jquery</a>`)
-        .appendTo(win.document.body)
-
-        // synchronous beforeunload
-        return $a.get(0).click()
-      })
-      .url().should('include', '/generic.html')
-      .window().then((win) => {
-        return new Promise((resolve) => {
-          const xhr = new win.XMLHttpRequest
-
-          xhr.open('GET', '/foo')
-          xhr.send()
-          xhr.onload = resolve
-        })
-      })
-      .wait('@getFoo').its('url').should('include', '/foo')
-    })
-  })
-
-  // FIXME: I have no idea why this is skipped, this test is rly old
-  context.skip('#cancel', () => {
-    it('calls server#cancel', function (done) {
-      let cancel = null
-
-      this.Cypress.once('abort', () => {
-        expect(cancel).to.be.called
-
-        done()
-      })
-
-      cy.server().then(function () {
-        cancel = cy.spy(cy.state('server'), 'cancel')
-
-        this.Cypress.trigger('abort')
-      })
-    })
+    cy.server()
   })
+})
 
-  // FIXME: I have no idea why this is skipped, this test is rly old
-  context.skip('#respond', () => {
-    it('calls server#respond', () => {
-      let respond = null
-
-      cy
-      .server({ delay: 100 }).then((server) => {
-        respond = cy.spy(server, 'respond')
-      })
-      .window().then((win) => {
-        win.$.get('/users')
+context('cy.route', () => {
+  it('throws error on use of cy.route', (done) => {
+    cy.on('fail', (err) => {
+      expect(err.message).to.equal('`cy.route()` was removed in Cypress version 12.0.0. Please update to use `cy.intercept()` instead.')
+      expect(err.docsUrl).to.equal('https://on.cypress.io/intercept')
 
-        return null
-      })
-      .respond().then(() => {
-        expect(respond).to.be.calledOnce
-      })
+      done()
     })
 
-    describe('errors', () => {
-      beforeEach(function () {
-        this.allowErrors()
-      })
-
-      it('errors without a server', (done) => {
-        cy.on('fail', (err) => {
-          expect(err.message).to.eq('cy.respond() cannot be invoked before starting the `cy.server()`')
-
-          done()
-        })
-
-        cy.respond()
-      })
-
-      it('errors with no pending requests', (done) => {
-        cy.on('fail', (err) => {
-          expect(err.message).to.eq('cy.respond() did not find any pending requests to respond to')
-
-          done()
-        })
-
-        cy
-        .server()
-        .route(/users/, {})
-        .window().then((win) => {
-          // this is waited on to be resolved
-          // because of jquery promise thenable
-          return win.$.get('/users')
-        })
-        .respond()
-      })
-    })
+    cy.route('/foo')
   })
+})
 
-  context('options immutability', () => {
-    it('does not mutate options for cy.server()', () => {
-      const options = { enable: false }
+context('Cypress.server.defaults', () => {
```

Comment:
Should we move these tests to `packages/driver/cypress/e2e/e2e/origin/commands/unsupported_commands.cy.ts`? "Unsupported commands" seems like an accurate description.

Reply:
I started to move these tests to unsupported_commands, but upon inspection, that file is for commands that are supported in cypress but not cy.origin. So instead I renamed this file to `removed_commands.cy.js` and moved the `Cypress.Cookies.defauts` and `Cypress.Cookies.preserveOnce` tests to this file.